server_extensions_extra/build.gradle

```diff
@@ -52,6 +52,7 @@ dependencies {
     compile(group: 'org.knowm.xchange', name: 'xchange-dvchain', version: '4.3.15')
     compile(group: 'org.knowm.xchange', name: 'xchange-coinbasepro', version: '4.3.15')
     compile(group: 'org.knowm.xchange', name: 'xchange-coingi', version: '4.3.18')
+    compile(group: 'org.knowm.xchange', name: 'xchange-enigma', version: '4.4.1')
```

Comment:
adding xchange-enigma:4.4.1 requires higher version of org.knowm.xchange:xchange-core please increase also version of other xchange-core dependent libraries otherwise it will break the other exchanges functionality. xchange-coinbasepro and others.